src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/DiscussionsConstants.java

```diff
@@ -0,0 +1,6 @@
+package com.wikia.webdriver.elements.mercury.components.discussions.common;
+
+public class DiscussionsConstants {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1118)